### PR TITLE
fix: nameet github issue number 757 and updates on  assets, driver validations 

### PR DESCRIPTION
**Pull request by nameetjambla**
In this file, I have updated the validation errors messages and patterns as per Rashmi testing and also solve GitHub issue number 757

### CODE REVIEW

File: src/app/pages/fleet/assets/add-assets/add-assets.component.html [hunk 1]
```diff
@@ -72,15 +72,14 @@ <h4 class="text-4 font-weight-bold mt-0 mb-0 text-dark">{{pageTitle}}</h4>
                           class="mandfield text-2 ml-1"><sup>*</sup></span> <i class="fas fa-exclamation-circle" data-toggle="tooltip"
                           data-placement="top" title="Vehicle Identification Number."></i></label>
                       <input type="text" [(ngModel)]="assetsData.VIN" name="VIN"
-                        placeholder="17-18 digits Alphanumeric , eg. 78678678678678689" class="form-control" maxlength="18" #vName="ngModel" required pattern="^[0-9\-]{17,18}$">
+                        placeholder="17-18 digits Alphanumeric , eg. 78678678678678689" class="form-control" maxlength="18" #vName="ngModel" required pattern="^[a-zA-Z0-9\-]{17,18}$">
```
Comment (kunalfleethawks) [inline src/app/pages/fleet/assets/add-assets/add-assets.component.html]: Please put valid VIN number in example : 2G1WH55K5Y9322458
You can check here https://vingenerator.org

File: src/app/pages/fleet/assets/add-assets/add-assets.component.html [hunk 2]
```diff
@@ -72,15 +72,14 @@ <h4 class="text-4 font-weight-bold mt-0 mb-0 text-dark">{{pageTitle}}</h4>
...
                         <div *ngIf="vName.invalid && (vName.dirty || vName.touched)" class="text-danger">
                           <div>
                             <div *ngIf="vName.errors.required">
                               VIN is required.
                             </div>
                             <div *ngIf="vName.errors.pattern">
-                               VIN contains 17-18 alphanumeric number only eg.78678678678678689
-                            </div>
+                               VIN contains 17-18 alphanumeric number and special characters for period('-') only eg.ARD12345-4er78                           </div>
```
Comment (kunalfleethawks) [inline src/app/pages/fleet/assets/add-assets/add-assets.component.html]: VIN number example https://vingenerator.org

**Review by kunalfleethawks — CHANGES_REQUESTED** [review]
Please address review comments

File: src/app/pages/fleet/assets/add-assets/add-assets.component.html (hunk 1 above)
Comment (nameetjambla) [reply src/app/pages/fleet/assets/add-assets/add-assets.component.html]: done

File: src/app/pages/fleet/assets/add-assets/add-assets.component.html (hunk 2 above)
Comment (nameetjambla) [reply src/app/pages/fleet/assets/add-assets/add-assets.component.html]: done

**Review by kunalfleethawks — APPROVED** [review]

**Pull request merged**